### PR TITLE
add error message to login and rewrite test

### PR DESCRIPTION
**Pull request by SetterVet**

### CODE REVIEW

**Comment by SetterVet** [comment]
image

**Comment by SetterVet** [comment]
fix position @rubycop @afurm

**Comment by afurm** [comment]
can you attach screenshot ? how it looks  after update.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
-  before_action :authenticate_user!
```
Comment (afurm) [inline app/controllers/application_controller.rb]: Why we remove it from here? don't we want all controllers/actions to be hidden from unauthorized users?

File: app/javascript/src/styles/layout/_devise_errors.scss [hunk 2]
```diff
@@ -40,3 +40,36 @@
     }
   }
 }
+
+/*flash*/
+.alert-error {
```
Comment (afurm) [inline app/javascript/src/styles/layout/_devise_errors.scss]: Can we use bootstrap alerts message style? https://getbootstrap.com/docs/4.0/components/alerts/

**Comment by SetterVet** [comment]
Fix @afurm @rubycop

**Pull request merged**